mlflow/ml-package-versions.yml

```diff
@@ -66,26 +66,23 @@ tensorflow:
       pip install tf-nightly
 
   models:
-    minimum: "2.3.0"
+    minimum: "2.6.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
       "== dev": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      pip install keras==`python -c 'import tensorflow; print(tensorflow.__version__)`
```

Comment:
If keras version does not match with tensorflow version, some error might happen such as https://github.com/mlflow/mlflow/runs/7944152032?check_suite_focus=true#step:12:43

So here I reinstall keras.

Reply:
What if `tensorflow` version is `x.y`, but `keras` doesn't have `x.y`?

Reply:
See https://pypi.org/project/keras/2.9.0 and https://pypi.org/project/tensorflow/2.9.1/

Reply:
Good point. I am trying installing keras with the same major.minor version, but ignore micro version.